src/blocks/venue/edit.js

```diff
@@ -212,6 +223,45 @@ const Edit = ({ attributes, setAttributes, isSelected }) => {
 							min={100}
 							max={1000}
 						/>
+						<PanelRow>
+							{__('Latitude / Longitude', 'gatherpress')}
+						</PanelRow>
+						<PanelRow>
+							<ToggleControl
+								label={
+									mapCustomLatLng
+										? __('Use custom values', 'gatherpress')
+										: __(
+												'Use default values',
+												'gatherpress'
+											)
+								}
+								checked={mapCustomLatLng}
+								onChange={(value) => {
+									setAttributes({ mapCustomLatLng: value });
+								}}
+							/>
+						</PanelRow>
+						{mapCustomLatLng && (
+							<>
+								<TextControl
+									label={__('Latitude', 'gatherpress')}
+									value={latitude}
+									onChange={(value) => {
+										Broadcaster({ setLatitude: value });
+										updateVenueMeta({ latitude: value });
+									}}
+								/>
+								<TextControl
+									label={__('Longitude', 'gatherpress')}
+									value={longitude}
+									onChange={(value) => {
+										Broadcaster({ setLongitude: value });
+										updateVenueMeta({ longitude: value });
+									}}
+								/>
```

Comment:
We should have a button that will "Reset" the coordinates. Just call the button "Reset" and have it take the address and run it through the API again to bring back original values.

Reply:
Seems that the changes in lat/long are not being retained. I did some updates, but I don't think anything that should have effected functionality. It changes, but then when I leave focus, it reverts... It might be good to use the redux stores rather than Broadcaster/Listener as that may be the issue.

Reply:
> Seems that the changes in lat/long are not being retained. I did some updates, but I don't think anything that should have effected functionality. It changes, but then when I leave focus, it reverts... It might be good to use the redux stores rather than Broadcaster/Listener as that may be the issue.

We implemented the store. So all these issues should be resolved. @mauteri

Reply:
@jmarx I would like to add the Reset button. I get what you are saying, but that might not be super obvious to people. A reset button would be.